### ISSUE TITLE
Fire up a repl

### ISSUE THREAD

**Issue by arichiardi** [eginez/huckleberry#2]
Hello @eginez :smile: !

I was trying to use `figwheel` for firing up a repl but did not go to far with it, I think the `-main` is missing (but I have created it and there is a compilation error:

```
Failed to compile "out/main.js" in 2.303 seconds.
----  Could not Analyze  out/cljs/core/async.cljs  ----

  java.lang.RuntimeException: No such var: ioc/state-machine, compiling:(cljs/core/async/macros.clj:18:19)

----  Analysis Error : Please see out/cljs/core/async.cljs  ----
```

How do you currently work on the project?

Thanks a lot! I hope I can contribute to this a bit :smile_cat:

**Comment by eginez** [comment]
It was basically driven by doo test, I haven't added a figwheel script, if you do it feel free to send a PR

**Comment by arichiardi** [comment]
Yes I did add it, but received that weird error back. I'll try to figure out what is going on..

On December 14, 2016 5:40:52 PM PST, Esteban <notifications@github.com> wrote:
>It was basically driven by doo test, I haven't added a figwheel script,
>if you do it feel free to send a PR
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/eginez/huckleberry/issues/2#issuecomment-267212175

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by arichiardi** [comment]
Solved with #3
